recipes/libpng/all/conanfile.py

```diff
@@ -37,6 +37,10 @@ class LibpngConan(ConanFile):
         "api_prefix": "",
     }
 
+    @property
+    def _is_clang_cl(self):
+        return str(self.settings.compiler) in ["clang"] and str(self.settings.os) in ['Windows']
```

Comment:
this condition is fragile and breaks mingw clang. This is why it's not widely used in CCI recipe.